rsrc.go

```diff
@@ -125,7 +81,7 @@ void ·get_NAME(Slice a) {
 	return nil
 }
 
-func run(fnamein, fnameico, fnameout, arch string) error {
+func Run(fnamein, fnameico, fnameout, arch string) error {
```

Comment:
1. Please rename this func to `Embed`, and move the `fnameout` and `arch` parameters to first & second position.
2. For consistency, please also rename `rundata` to `embeddata` and also move `fnameout` + `arch` parameters in the same way as mentioned above.
3. [extra] It would be super cool if you could try writing a godoc for the newly exported `Embed` func. But I'll leave this as optional, and also I must make it clear that if you try I expect we'll be doing quite a lot more editing on it before I'm satisfied with the wording — and I don't want to put too much burden on you here.